src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,14 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Country implements Cloneable {
```

Comment:
Implementing `Cloneable` and providing a `clone` method is not typical for entity classes. Entity instances are usually managed by an ORM framework like Hibernate, which handles their lifecycle, and cloning entities can lead to unexpected behavior and issues with identity and state management. Unless you have a specific reason that requires entity cloning, you should remove the `Cloneable` interface and the `clone` method.